src/features/applications/models/user__has_direct_access_to__application.ts

```diff
@@ -0,0 +1,126 @@
+import type {
+	AbstractSqlModel,
+	Relationship,
+	SelectQueryNode,
+} from '@balena/abstract-sql-compiler';
+import * as _ from 'lodash';
+
+export const addToModel = (
+	abstractSql: AbstractSqlModel,
+	selectQueryNodeTransformer: (
+		selectQueryNode: SelectQueryNode,
+	) => SelectQueryNode = _.identity,
+) => {
+	abstractSql.tables['user-has direct access to-application'] = {
```

Comment:
I see though that someone could argue against the `direct` part of the name, since in oB users do not have a direct link to any app :thinking: